src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.FSharp.xaml

```diff
@@ -33,6 +33,26 @@
                 HelpUrl="https://docs.microsoft.com/dotnet/fsharp/language-reference/compiler-options"
                 Category="General" />
 
+  <EnumProperty Name="FSharpPreferNetFrameworkTools"
+              DisplayName="Framework tools"
+              Description="Compiler to use when building the project"
+              Category="General">
+    <EnumValue Name="true"
+             DisplayName=".NET Framework" />
+    <EnumValue Name="false"
+             DisplayName=".NET SDK" />
+  </EnumProperty>
+
+  <EnumProperty Name="FSharpPrefer64BitTools"
+            DisplayName="Prefer 64 bit tools"
+            Description="Use a 64 bit or 32 bit version of the compiler when available"
```

Comment:
I suggest making this text match the "Prefer 32-bit" property, and adding a trailing full stop.

```suggestion
            Description="Use a 64-bit compiler on systems that support both 32-bit and 64-bit compilers."
```